Topics/Software_Engineering/Scrum Planning Meeting.md

```diff
@@ -0,0 +1,32 @@
+# Scrum Planning Meeting
+
+## Introduction
+In the Scrum framework, sprints function as short, consistent cycles during which a set amount of work is completed. Since they are the heartbeat of the framework, ensuring sprints are well planned, or ensuring a successful Sprint Planning Meeting, is a crucial part of the framework.
+
+
+## What does a Sprint Planning Meeting Encompass? 
+The meeting is held with the entire Agile team (including the ScrumMaster and Product Owner) and aims to conclude with a set of agreed upon **Product Backlog** items to be completed for the current sprint commitment. The **Product Backlog** is an ordered list of tasks essential for improving the product and satisfying the stakeholders, while the **Sprint Backlog** is a list of tasks selected from the Product Backlog to be completed during a specific sprint.
+
+The ScrumMaster is responsible for leading the Planning Meeting and the Product Owner is responsible for detailing the Product Backlog items and ensuring alignment with the product’s goals. The planning addresses the value the sprint can bring to the stakeholders and the team collaborates to define a *Sprint Goal* before the end of the Sprint Planning meeting. Following, upon discussion with the Product Owner, the developers select items from the Product Backlog to include in the Sprint Backlog and determine this based on their prior and future Sprint performance. Additionally, for each Sprint Backlog item that is selected, the developers on the team decide how to break down and complete tasks for the sprint based on the team’s definition of “Done”.
+
+
+## Integrating Sprint Planning Meetings in CSC301 
```

Comment:
I like how you incorporated an example of how to directly use scrum planning meeting in the course! This will definitely help other students get an idea of how this strategy can be put into practice.

Reply:
glad to hear!